app/settings.py

```diff
@@ -19,9 +20,15 @@ def parse_env_bool(key: str, default: str = "false") -> bool:
     return os.getenv(key, default).lower() in ("yes", "true", "1", "on")
 
 
+def parse_env_str(key: str, default: str | None = None) -> str | None:
+    val = os.getenv(key, default)
+    return val if val else None
```

Comment:
This is done for coalescing an empty string into `None`

Reply:
I think it would be better to use `default` instead of `None`, if we consider empty string same as key not found
nit: you can use `value or default` to simplify that check